.github/workflows/release.yml

```diff
@@ -0,0 +1,39 @@
+# References:
+# - https://github.com/cli/cli/blob/v2.32.1/.github/workflows/deployment.yml
+# - https://github.com/goreleaser/goreleaser-action/tree/v4.4.0#usage
+
+name: Release
+
+permissions:
+  contents: write
```

Comment:
This is the default permission.
Its not needed. 
Excepf of the case, that now all other permissions are set to `none`... Not sure if we want to this 🤷 

See also https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs#defining-access-for-the-github_token-scopes

Reply:
I really have not much idea here I just took what the references indicated. Did you checked those references? I mean they are two very valid and credible sources.

Reply:
Okay, let me give you a bit more explanation about this:

In each GitHub Action you get a `GITHUB_TOKEN` as a env. variable.
In the beginning of GH Actions you could literally anything with that.
Hell, even forks could used this to modify **your code** or, well, doing some other evil stuff.

So GitHub did some changes on that.
One thing was that forks can't use this token anymore. Or at least only limited actions (if I remember correctly).

Another solution to make it more robust, and espeically that other actions (so the stuf we use in "`use:`") dont' do some evil stuff, they introduced this permissions thingy.

The permissions thing here is only there to control the `GITHUB_TOKEN`.
By default, so in case you don't specify anything, the `GITHUB_TOKEN` has read & write acces to "stuff" (kinda of everything).

You can change this. By just specifying what permissions you give to this token.
Any permissions that are not listed here are set to "none".
Not 100% sure what dthis means, but I think its like "neither read NOR write" access.

Coming back to this config now:
You only set `contents: write`.
This is, at the first look, the default *anyways*. By default, the `GITHUB_TOKEN` **has already** write access to contents.
However, this makes stuff a bit more secure because this will set all other permission to `none`.

For example, none of our actions here can't post `issues`, or modyfing `pull_request`... Or something else 🤷 😁 

So, I'm not sure we want this 🤷 
Of course it is a bit security related. But having such things in the code base also leads to questions in the future. 
Like, we have right now 😁 

So my questions: Do we want this?
We have `contents: write` access anyways. So in case we remove it, everything still works as expected...

Reply:
@StefMa first of all thanks for detailed explanation here. For me it was simple, I looked at a 'working reference': https://github.com/cli/cli/blob/v2.32.1/.github/workflows/deployment.yml#L7 because if that works then our changes should work too. Did you looked at it? Also if there are no other permissions that it is also okay right? For this workflow we do not need other permissions. Still if you think this should be removed then I will do it. Anyways there is no good way to test workflows :shrug:

Reply:
Nah, it's fine for me 👌 🙃